target/linux/ramips/rt305x/base-files/etc/board.d/01_leds

```diff
@@ -66,6 +66,12 @@ asiarf,awapn2403)
 dlink,dcs-930l-b1)
 	ucidef_set_led_netdev "wifi" "WiFi" "$boardname:blue:wps"
 	;;
+dlink,dir-506l)
+	ucidef_set_led_gpio "lo_bat" "Low battery" "${boardname}:red:status" "10" "0"
+	ucidef_set_led_default "status" "Status" "${boardname}:green:status" "1"
+	ucidef_set_led_netdev "lan" "LAN" "${boardname}:green:usb" "eth0" "tx rx"
```

Comment:
Either it's an USB LED, or it's an ethernet LED. Just stick to what the vendor does.

Reply:
I fail to find usb trigger for this platform, so I make LED with USB logo to show lan activity.

Reply:
That shouldn't be done for default setup. If it's an USB LED, it's should indicate USB. If you can't manage that, just leave it unassigned. The user then can assign what he likes afterwards.